widget/embedded/src/components/Layout/Layout.styles.ts

```diff
@@ -8,8 +8,10 @@ export const Container = styled('div', {
   overflow: 'hidden !important',
   boxShadow: '15px 15px 15px 0px rgba(0, 0, 0, 0.05)',
   width: '95vw',
+  minWidth: '300px',
   maxWidth: '390px',
   maxHeight: '700px',
+  textAlign: 'left',
```

Comment:
@Ikari-Shinji-re 
Please check your changes with something this to make sure every thing works. I think we need more changes in parent container rather than just adding a text-align for the  parent.

```css
*,
div *,
span *,
input * {
    text-align: right;
}
```